.github/workflows/heroku.yml

```diff
@@ -47,7 +52,7 @@ jobs:
         git fetch --prune --unshallow
         git push --force \
           https://user:$HEROKU_API_KEY@git.heroku.com/${{ inputs.heroku-app }}.git \
-          ${{ github.event.repository.default_branch }}
+          ${{ inputs.branch }}
```

Comment:
This doesn't seem to work, see my test at https://github.com/84codes/dev-playground84/actions/runs/11457537361/job/31877975220#step:4:14

Reply:
Hm, taking a look 👀

Reply:
ah yes, it should still be pushed to `main` on Heroku's end?

Reply:
https://devcenter.heroku.com/articles/git#deploy-from-a-branch-besides-main like this

Reply:
> https://devcenter.heroku.com/articles/git#deploy-from-a-branch-besides-main like this

bingo!

fixing now...

Reply:
That's a nice way to test workflows https://github.com/84codes/dev-playground84/commit/bc3486c1288f422e077cbf35b005b71b4867d834 👍

Reply:
Sorry missed you started to chat here, I pushed https://github.com/84codes/actions/commit/294d5993c0adffdcecba9d6508e1d633a392d5f0

Reply:
Ah right we need to checkout the right stuff too, I had this thought ...

Reply:
Ah, ok. Thanks for doing the proper QA and fixing it!

Good to know I can use dev-playground for testing out workflows in the future.

Reply:
Right, this one https://github.com/actions/checkout?tab=readme-ov-file#checkout-a-different-branch 💡

Reply:
Sorted with https://github.com/84codes/actions/commit/d847bb6b9435c706e515f51a8c9f38c683031046

Reply:
gj!